cli.js

```diff
@@ -158,7 +158,10 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
```

Comment:
`process.stdout.write('');` was there before me, @sindresorhus should have a reason for it.